app/models/mtUser.js

```diff
@@ -88,6 +90,13 @@ mtUserSchema.plugin(linz.formtools.plugins.document, {
                 label: 'Customised edit form',
             },
         ],
+        export: [
+            {
+                exclusions: '_id',
+                label: 'Choose fields to export',
+                dateFormat: 'DD MMM YYYY',
```

Comment:
@allanchau, what does this line do?

Reply:
There is code in Linz that allows you to format dates using `dateFormat`. 🤷‍♂️ I don't remember when it was added, but it does seem to work.